src/app/services/actions/permissions-action-creator.ts

```diff
@@ -81,13 +89,17 @@ export function fetchScopes(): Function {
       const response = await fetch(permissionsUrl, options);
       if (response.ok) {
         const scopes = await response.json();
-        return dispatch(
-          fetchScopesSuccess({
-            hasUrl,
-            scopes
-          })
-        );
+
+        return permissionsPanelOpen ? dispatch(fetchFullScopesSuccess({
+          hasUrl: false,
```

Comment:
since there is an explicit name for the scopes, there would be no need for the hasUrl flag